slapp/transforms/image_utils.py

```diff
@@ -0,0 +1,63 @@
+from typing import Tuple
+from pathlib import Path
+from PIL import Image, ImageDraw, ImageFont
+from math import ceil
+
+
+def add_scale(image_path: Path,
```

Comment:
Would be nice to overload this and pass either a path or the image data

Reply:
It's possible to do but it will be somewhat ugly, I tried this and you can make a PIL Image object from a numpy array but you can't draw or write to it. So I would have to make a temp dir to save the numpy array as a png to reload it into PIL as a png. The functionality to go from numpy -> Image in PIL (that has all features) is not there unfortunately.

Reply:
But a PIL image should be fine, right?

Reply:
It is I made a mistake and didn't convert the numpy array to RGBA when loading so drawing was failing.